src/libs/PolicyUtils.js

```diff
@@ -0,0 +1,63 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+
+/**
+ * Checks if we have any errors stored within the POLICY_MEMBER_LIST. Determines whether we should show a red brick road error or not.
+ * Data structure: {email: {role:'bla', errors: []}, email2: {role:'bla', errors: [{1231312313: 'Unable to do X'}]}, ...}
+ *
+ * @param {Object} policyMemberList
+ * @returns {Boolean}
+ */
+function hasPolicyMemberError(policyMemberList) {
+    return _.some(policyMemberList, member => !_.isEmpty(member.errors));
+}
+
+/**
+ * The policy has an error if there are errors under the 'errors' key or the 'errorFields' key.
+ *
+ * @param {Object} policy
+ * @param {Object} policy.errors
+ * @param {Object} policy.errorFields
+ * @return {Boolean}
+ */
+function hasPolicyError(policy) {
+    const errors = lodashGet(policy, 'errors', {});
+    const errorFields = lodashGet(policy, 'errorFields', {});
+    const hasFieldErrors = _.some(errorFields, fieldErrors => !_.isEmpty(fieldErrors));
+    return !_.isEmpty(errors) || hasFieldErrors;
```

Comment:
can we instead return fast on line 26-27 above `if !_.isEmpty(errors)` ?

Reply:
+1, early returns FTW

Reply:
Good idea 👍